### ISSUE TITLE
[flutter_auth_web]: Not working on iOS (?)

### ISSUE THREAD

**Issue by Ahmadre** [FirebaseExtended/flutterfire#1560]
**Describe the bug**
After importing flutter_auth and flutter_auth_web and I try to run it on iOS, I get following weird error:

```
Launching lib/main.dart on iPhone 11 Pro in debug mode...
CocoaPods' output:
↳
      Preparing
    Analyzing dependencies
    Inspecting targets to integrate
      Using `ARCHS` setting to build architectures of target `Pods-Runner`: (``)
    Finding Podfile changes
      A firebase_auth_web
      A firebase_messaging
      - Firebase
      - Flutter
      - apple_sign_in
      - firebase_auth
      - firebase_core
      - firebase_storage
      - google_sign_in
      - path_provider
      - url_launcher
    Fetching external sources
    -> Fetching podspec for `Flutter` from `Flutter`
    -> Fetching podspec for `apple_sign_in` from `.symlinks/plugins/apple_sign_in/ios`
    -> Fetching podspec for `firebase_auth` from `.symlinks/plugins/firebase_auth/ios`
    -> Fetching podspec for `firebase_auth_web` from `.symlinks/plugins/firebase_auth_web/ios`
    [!] No podspec found for `firebase_auth_web` in `.symlinks/plugins/firebase_auth_web/ios`
    /usr/local/Cellar/cocoapods/1.8.3/libexec/gems/cocoapods-1.8.3/lib/cocoapods/external_sources/path_source.rb:14:in `block in fetch'
    /usr/local/Cellar/cocoapods/1.8.3/libexec/gems/cocoapods-1.8.3/lib/cocoapods/user_interface.rb:64:in `section'
    /usr/local/Cellar/cocoapods/1.8.3/libexec/gems/cocoapods-1.8.3/lib/cocoapods/external_sources/path_source.rb:11:in `fetch'
    /usr/local/Cellar/cocoapods/1.8.3/libexec/gems/cocoapods-1.8.3/lib/cocoapods/installer/analyzer.rb:901:in `fetch_external_source'
    /usr/local/Cellar/cocoapods/1.8.3/libexec/gems/cocoapods-1.8.3/lib/cocoapods/installer/analyzer.rb:880:in `block (2 levels) in fetch_external_sources'
    /usr/local/Cellar/cocoapods/1.8.3/libexec/gems/cocoapods-1.8.3/lib/cocoapods/installer/analyzer.rb:879:in `each'
    /usr/local/Cellar/cocoapods/1.8.3/libexec/gems/cocoapods-1.8.3/lib/cocoapods/installer/analyzer.rb:879:in `block in fetch_external_sources'
    /usr/local/Cellar/cocoapods/1.8.3/libexec/gems/cocoapods-1.8.3/lib/cocoapods/user_interface.rb:64:in `section'
    /usr/local/Cellar/cocoapods/1.8.3/libexec/gems/cocoapods-1.8.3/lib/cocoapods/installer/analyzer.rb:878:in `fetch_external_sources'
```

**Expected behavior**
Flutter Auth Web should not try to pull a cocoapod...

**Comment by Ahmadre** [comment]
I solve it by deleting ../.flutter-plugins entry for flutter_auth_web. Maybe someone here can disable that symlink

**Comment by Ahmadre** [comment]
I solved it by using firebase dart implementation.

**Comment by emailsubjekt** [comment]
> I solved it by using firebase dart implementation.

Can you please elaborate more on this and kindly provide a code example. Thank you

**Comment by hterkelsen** [comment]
I need to add a fake podspec to `firebase_auth_web` so we don't break iOS. I will do that today

**Comment by Ahmadre** [comment]
@emailsubjekt Sure :), just follow this repo:

https://github.com/FirebaseExtended/firebase-dart

and use this pub:

https://pub.dev/packages/firebase

I solved by separating my project in: VM and WEB project. Before that I got a boolean: isBrowser with which I identified, if Flutter is running on web or vm. But I can't use this package neither, because it broke not only iOS but also Android. And I need solutions now, because we use it in production. But I got tons of solutions. At the end I would build my own wrapper, but that's not the solution ^^

By the way: Thank you sooo much flutterfire team for you packages here for VM! I love them :)

_Gesendet von meinem Pixel 2 XL mit [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_
